webbpsf/optics.py

```diff
@@ -388,7 +388,7 @@ def __init__(
         if which == 'LLNL':
             raise NotImplementedError('Rotated field mask for LLNL grism not yet implemented!')
         elif which == 'Bach':
-            transmission = os.path.join(utils.get_webbpsf_data_path(), 'NIRISS/optics/MASKGR700XD.fits.gz')
+            transmission = os.path.join(utils.get_webbpsf_data_path(), 'NIRISS/optics/MASKGR700XD.fits.gz')  # TODO - Unused value, delete entire statement?
```

Comment:
Probably, but I'll defer to @obi-wan76 for this since he's going to be revising the GR700XD models at some point.